src/utils/rescale.ts

```diff
@@ -0,0 +1,11 @@
+import { DoubleArray } from 'cheminfo-types';
+
+export function rescale(data: DoubleArray, factor: number) {
```

Comment:
This is the kind of method that should be in ml-spectra-processing but actually it already exists

https://github.com/mljs/spectra-processing/blob/master/src/x/xMultiply.ts